packages/ng/user/picture/user-picture.component.html

```diff
@@ -1,3 +1,11 @@
-<div class="picture" [ngStyle]="style">
-	<span class="userPicture-initials" *ngIf="!hasPicture">{{ initials }}</span>
-</div>
+<ng-container *ngIf="!!hasPicture; then displayPicture; else displayInitials"></ng-container>
+
+<ng-template #displayPicture>
+	<img class="picture" [src]="pictureHref" [alt]="initials" (error)="pictureError()" />
+</ng-template>
```

Comment:
I think we used to use a background image because it allowed us to center the image in the element. We can do this with `object-fit` and `object-position` on the `img` element now.

Reply:
Hi @shajz,

1.
Your suggestion to simplified with `*ngIf="hasPicture; else displayInitials"` works perfectly. I just used the more "detailled" version as we do in Poplee Training.

2.
Regarding the `ngSrc` ([Angular doc](https://angular.io/api/common/NgOptimizedImage)), it would in fact be better, but we have to add width and height even if the SCSS specifies the image dimensions :
```html
<img class="picture" [ngSrc]="pictureHref" [alt]="initials" (error)="pictureError()" width="40" height="40" />
``` 
With fill attribute, it overrides the SCSS dimensions :
![image](https://github.com/LuccaSA/lucca-front/assets/97245071/d7449be0-308c-4c30-ba2f-05e0c8504d59)

So, do we use the `ngSrc` attribute ?

Reply:
I don't understand why not keeping it simple ? The detailed version adds boilerplate and doesn't help with readability.
Also we don't need to use `!!hasPicture` since it's a boolean, `hasPicture` is fine :)

And let's not use ngSrc, we don't want breaking changes and force people to force a width/height here :)

Reply:
To my mind, the split of if / then / else improves a bit the readability, even if I'm ok with both options. Moreover, this debate will soon be a thing of the past with the new control flow. 😉 

I will :
* update with if / else
* remove the unnecessary `!!` (well seen)
* keep the `src` attribute